### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Apatakiti**
> **Modify the Awesome books application to have:**

-  A Navigation bar.
-  Three content sections:

1.  Books list.
2.  Add book form.
3.  Contact info.

- Make sure that when you click on any of the links, the main area changes to display only one of the three content sections.
- To change the contents of the page, don't use any library or framework, use only plain JavaScript and CSS.

I think I have done all the necessary procedure

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// import { Datetime } from './luxon.js';
```
Comment (codecaiine) [inline app.js]: - [ ] Good job with your [project implementation](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) :smile:  But there are some issues you need to address. From the project requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install) to display the current date below the navigation bar. Follow this article on how  [get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Please fix it cause this is very important in your developer job you will probably be asked to use external tools :+1:  

Screenshot from 2022-08-05 00-50-18

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Ademola-coding @RodChak , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive Pull Request :heavy_check_mark: 
- Good use of gitflow :heavy_check_mark: 
- Clean code :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (Apatakiti) [reply app.js]: Done

**Review by keroloussamy — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**